test/_common.py

```diff
@@ -68,6 +68,7 @@ def item(lib=None):
         genre=u'the genre',
         lyricist=u'the lyricist',
         composer=u'the composer',
+        composer_sort=u'the sortname of the composer',
```

Comment:
There's no `artist_sort` here AFAICT, are you sure `composer_sort` should be here?